cypress/e2e/test.spec.ts

```diff
@@ -0,0 +1,73 @@
+import basketPage from "../pages/basketPage";
+import modifyOrderPage from "../pages/modifyOrderPage";
+import orderNavigationPage from "../pages/orderNavigationPage";
+import orderPastaPage from "../pages/orderPastaPage";
+import orderPizzaPage from "../pages/orderPizzaPage";
+
+describe("Testing pizza hut application", () => {
+	before(() => {
+		orderNavigationPage.pizzaMenuButton.click();
+	});
+
+	it("Should select and verify item", () => {
+		orderPizzaPage.pizzaCards.eq(0)
+			.within(() => {
+				orderPizzaPage.pizzaNameInCard
+					.then($element => {
+						cy.wrap($element.contents()[0].textContent).as("pizzaName")
+					});
+				orderPizzaPage.pizzaPriceInCard
+					.invoke("text")
+					.as("pizzaPrice");
+				orderPizzaPage.pizzaAddButtonInCard
+					.click();
+			});
+
+		cy.getAliases(["@pizzaName", "@pizzaPrice"]).then(([name, price]) => {
+			basketPage.subTotalValue
+				.should("contain.text", price);
+
+			basketPage.basketItems.eq(0).within(() => {
+				basketPage.basketItemName.should("contain.text", name);
+				basketPage.basketItemPrice.should("have.text", price);
+			});
+		});
+	});
+
+	it("Should modify and verify item", () => {
+		basketPage.basketItems.eq(0).within(() => {
+			basketPage.basketItemModifyOrderButton.click();
+		});
+
+		modifyOrderPage.addToppingButton.eq(0).click();
+		modifyOrderPage.modifiedPrice.then(($span) => {
+			cy.wrap($span.contents()[1].textContent).as("pizzaPrice");
+		});
+		modifyOrderPage.addToBasketButton.click();
+
+		cy.get("@pizzaPrice").then((price) => {
+			basketPage.subTotalValue
+				.should("contain.text", price);
+		});
```

Comment:
Please explain and am guessing you are removing first product found can we keep specific data which needs to be removed.

Reply:
I am not using testIsolation therefore it is continuation of previous test. So, I am removing first one available. Also, there is a possibility that pizza I am targeting may not be available at certain time. So, it is a generic scenario.